clean-abap/CleanABAP_es.md

```diff
@@ -0,0 +1,5127 @@
+
+1. Convence al equipo. Comunica y explica el nuevo estilo y logra que todos en el equipo
+lo acepten. No necesitas comprometerte con toda la guía desde el inicio, solo comienza
+con un sub-conjunto de reglas indiscutibles y evoluciona a partir de eso. 
+
+2. Sigue la regla de los _boy scout_ en tu rutina de trabajo diaria:
+_siempre deja el código un poco más limpio de lo que lo encontraste_.
+No te obsesiones dedicando horas en "limpiar el campamento", solo
+dedica un par de minutos adicionales y observa como las mejoras se acumulan en el tiempo.
+
+3. Construye _islas limpias_: de vez en cuando, elige un pequeño objeto o componente y trata
+de hacerlo limpio en todos los sentidos. Estas islas demuestran el beneficio de lo que estás
+haciendo y forman una base solidamente probada para hacer más refactorizaciones en el futuro.
+
+4. Habla de ello. No importa si usas [Revisiones de código Fagan](https://en.wikipedia.org/wiki/Fagan_inspection) como antaño, llevas a cabo sesiones informativas o formas paneles de discusión en tu herramienta de chat favorita: 
+vas a necesitar hablar de tus experiencias y aprendizajes, para habilitar que el equipo llegue
+a un entendimiento común.
+
+### Cómo revisar automáticamente
+
+> [Clean ABAP](#clean-abap) > [Contenido](#contenido) > [Introducción](#introducción) > [Esta sección](#cómo-revisar-código-automáticamente)
+
+[code pal para ABAP](https://github.com/SAP/code-pal-for-abap)
+provee una suite comprensiva de revisiones automáticas para Clean ABAP.
+
+ABAP Test Cockpit, Code Inspector, Extended Check y Checkman proveen algunas revisiones
+que te pueden ayudar a detectar ciertos problemas.
+
+[abapOpenChecks](https://github.com/larshp/abapOpenChecks),
+es una colección Open Source de revisiones de Code Inspector,
+también cubre algunos anti-patrones.
+
+[abaplint](https://github.com/abaplint/abaplint) es una reimplementación Open Source del parser de ABAP. Funciona
+sin un sistema SAP y está pensado para ser utilizado de manera serializada junto con abapGit. Ofrece múltiples integraciones (GitHub Actions, Jenkins, editores de texto...), cubre algunos de los anti-patrones y puede ser usado también para revisar formato y convenciones de código.
+
+### Cómo se relaciona con otras guías
+
+> [Clean ABAP](#clean-abap) > [Contenido](#contenido) > [Introducción](#introducción)> [Esta sección](#cómo-se-relaciona-con-otras-guías)
+
+Nuestra guía sigue el _espiritú_ de Clean Codem
+lo que significa que hicimos algunos ajustes para el lenguaje de programación ABAP,
+por ejemplo [Lanza CX_STATIC_CHECK para excepciones que se pueden manejar](#lanza-cx_static_check-para-excepciones-que-se-pueden-manejar).
+
+Algunos hechos provienen de la [Guía para programación ABAP](https://help.sap.com/doc/abapdocu_751_index_htm/7.51/en-US/index.htm?file=abenabap_pgl.htm),
+con la cual esta guía es compatible en su mayor parte. Las desviaciones a esa guía siempre están indicadas
```

Comment:
"esta" with an accent is used when a subject is being indirectly referred to, which is not the case here as it has "guía" right after it.  Also, it has not been necessary to accentuate it at all for a while now:
[News article](https://www.lavanguardia.com/cultura/ortografia/20200227/473613492179/este-ese-aquel-acentuacion-adjetivos-pronombres.html)
[RAE](https://www.rae.es/dpd/tilde#321) 

"la" is also correct, since it is referencing the "Guía para programación ABAP" which is a feminine.

I agree that using "respecto" better conveys the meaning of the next sentence.

Reply:
I am not fully sure about the accent on `esta` so please leave it out.

Regarding the phrase `con lo cual`, right, I understood the sentence wrong, having the link in there cased confusion, since to be able to add comments I am reading it in markdown directly.

Reply:
Yeah, making sense of writing in markdown is tough when there are links.

This is the edited version that will be committed.

```suggestion
con la cual esta guía es compatible en su mayor parte. Las desviaciones respecto a esa guía siempre están indicadas
```